go.mod

```diff
@@ -106,3 +106,7 @@ require (
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/gballet/go-verkle => github.com/gballet/go-verkle v0.0.0-20230215214211-703246ecfb1f
+
+replace github.com/crate-crypto/go-ipa => github.com/jsign/go-ipa v0.0.0-20230215213932-c52c461f8ea7
```

Comment:
Use the tenative `go-verkle` and `go-ipa` with the optimizations.
Whenever (_if_...) that is merged, we need to remove this.